articles/connections/database/custom-db.md

```diff
@@ -1,52 +1,79 @@
+1. Navigate to [Connections > Database](${manage_url}/#/connections/database) on the Auth0 dashboard.
+2. Click the **+ Create DB Connection** button.
+3. Provide a name for the database and configure the available options.
+
+![](/media/articles/connections/database/database-connections.png)
+
+3. Navigate to the **Custom Database** tab.
+4. Toggle on the **Use my own database** switch.
 
 ![](/media/articles/connections/database/custom-database.png)
 
-## 3. Provide action scripts
+## Create database action scripts
+
+Create scripts in the **Database Action Scripts** section to configure how authentication works when using your database.
 
-You have to provide a login script to authenticate the user that will execute each time a user attempts to log in. Optionally, you can create scripts for sign-up, email verification, password reset and delete user functionality.
+A Login script is required. Additional scripts for user functionality, such as password resets, are optional. You can write your own scripts or select a template from the **Templates** dropdown and adjust it to your requirements.
+
+The available actions are:
+
+Name | Description | Parameters
+-------|-------------|-----------
+Login (Required) | Executes each time a user attempts to log in. | `email`, `password`
+Create | Executes when a user signs up. | `user.email`, `user.password`
+Verify | Executes after a user follows the verification link. | `email`
+Change Password | Executes when a user clicks on the confirmation link after a reset password request. | `email`, `newPassword`
+Get User | Retrieves a user profile from your database without authenticating the user. | `email`
+Delete | Executes when a user is deleted from the API or Auth0 dashboard. | `id`
 
 ::: note
-When creating users, Auth0 calls the **Get User** script before the **Create** script. Be sure that you have implemented both. Refer to the [Dashboard](${manage_url}) for sample scripts.
+When creating users, Auth0 calls the **Get User** script before the **Create** script. Be sure that you have implemented both.
 :::
 
-These custom scripts are *Node.js* code that run in the tenant's sandbox. Auth0 provides templates for most common databases, such as: **ASP.NET Membership Provider**, **MongoDB**, **MySQL**, **Oracle**, **PostgreSQL**, **SQLServer**, **Windows Azure SQL Database**, and for a web service accessed by **Basic Auth**. Essentially, you can connect to any kind of database or web service with a custom script.
+### Create the Login script
 
-This article shows how to implement the login script for **MySQL**. If you want a sample script for another action (sign-up, password reset, etc) or another database, use the dropdown options at the [Dashboard](${manage_url}).
+Create the required Login script to authenticate users. The Login script will run each time a user attempts to log in. You can write your own Login script or select a template from the **Templates** dropdown.
```

Comment:
The first sentence seems redundant `Create the required Login script to authenticate users.`